### PR TITLE
[Merged by Bors] - Initial stuff for crates.io publishing

### PR DESCRIPTION
**Pull request by simlay**
Subtask for #1450.

### CODE REVIEW

File: src/spu-schema/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio-spu-schema"
-version = "0.7.0"
+version = "0.8.0"
```
Comment (sehz) [inline src/spu-schema/Cargo.toml:3]: does it require major bump?

**Review by sehz — CHANGES_REQUESTED** [review]

File: src/spu-schema/Cargo.toml (hunk 1 above)
Comment (simlay) [reply src/spu-schema/Cargo.toml:3]: I think so, because `wasm_payload` was added to `StreamFetchRequest` in https://github.com/infinyon/fluvio/pull/1327

Comment (sehz) [reply src/spu-schema/Cargo.toml:3]: k

**Review by sehz — APPROVED** [review]

**Pull request closed without merging**